test/fork/deployment/PostDeployment.t.sol

```diff
@@ -24,12 +24,14 @@ import { Initializable } from "@openzeppelin/contracts-upgradeable/proxy/utils/I
 
 contract CSModuleDeploymentTest is Test, Utilities, DeploymentFixtures {
     DeployParams private deployParams;
+    uint256 adminsCount;
 
     function setUp() public {
         Env memory env = envVars();
         vm.createSelectFork(env.RPC_URL);
         initializeFromDeployment(env.DEPLOY_CONFIG);
         deployParams = parseDeployParams(env.DEPLOY_CONFIG);
+        adminsCount = block.chainid == 1 ? 1 : 2;
```

Comment:
you can get it by cheching the zero address of `deployParams.secondAdminAddress`

Reply:
but chainId is also ok

Reply:
Let's leave it for now